common/pom.xml

```diff
@@ -56,7 +56,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>114</maxAllowedViolations>
+          <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
@vrozov I checked with logViolationsToConsole as true, violations are printed

Please refer to this document.
https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html
checkstyle.console is the user property for logViolationsToConsole. In our case it is set to false.

consoleOutput will output violations + warnings. The user property associated with that is checkstyle.consoleOutput.